keyboards/2x5keypad/2x5keypad.h

```diff
@@ -0,0 +1,23 @@
+#ifndef KB_H
+#define KB_H
+
+#include "quantum.h"
+#include "led.h"
+
+#define RED_LED D0
+#define BLUE_LED B5
+#define GREEN_LED B6
+
+
+#define KEYMAP( \
```

Comment:
We've switched from using the "keymap" working, to "layout"

```suggestion
#define LAYOUT( \
```